pkg/config/template.go

```diff
@@ -47,3 +64,74 @@ func getTemplates(templatePath string) (CiTemplates, error) {
 func isYAML(file string, info os.FileInfo) bool {
 	return !info.IsDir() && (filepath.Ext(file) == ".yaml" || filepath.Ext(file) == ".yml")
 }
+
+// TemplateCMController holds the details needed for the configmap controller
+type TemplateCMController struct {
```

Comment:
`"controller"` has a specific meaning, and this is not a controller ;)

Reply:
In the beginning, I wanted to make it as a controller, and I stacked with the name :). I will rename that.